app/scripts/modules/core/src/application/nav/verticalNav.less

```diff
@@ -0,0 +1,71 @@
+.nav-section {
```

Comment:
not a must-do suggestion — and not even something you have to adhere to generally — but I usually have a preference for leveraging our [spacing](https://github.com/spinnaker/styleguide/blob/master/src/styles/spacing.css) and [flex layout](https://github.com/spinnaker/deck/blob/master/app/scripts/modules/core/src/presentation/flex-layout.less) utility classes in JSX vs. encoding margin, padding, and flex rules into stylesheets.

In my opinion (again not authoritative) working with layout and spacing within the component markup is much simpler to reason about than trying to mentally map layout rules to JSX elements based on their classnames (especially when you're not familiar with a component's code or the class names are not self-evident). If you're curious here's a [recent example](https://github.com/spinnaker/deck/pull/8157/files#diff-b42fb40803c15e767e7fe6881f22f1c1R18-R36) of what their usage looks like.

Mostly just bringing up for awareness, I have no strong feelings about this PR's use of layout rules in particular.

Reply:
That is fair. I have a different opinion coming from my old best practices where CSS is more closely tied to each component. I prefer it to be all consolidated in one place, and reused styles can be inherited from the parent class. With the spacing/layouts, components require multiple classes, many with only 1 attribute. This means the style is scattered about making it a hunt to modify styling and determine inheritance. 

That being said, since most of the team is used to the pattern you described I can make some adjustments. I will have another PR coming out soon that consolidates the nav components in the vertical navbar. This will tie everything together so I think it makes the most sense to update it there where I will already be putting final touches on the styles.